tests/test_fuzz.py

```diff
@@ -77,7 +77,14 @@
 
 
 def onlyIfHasHypothesis(has_hypothesis):
-	"""Decorator to handle optional loading."""
+	"""
+	Conditionally enable a class based on the availability of the hypothesis library.
+	If the provided flag is False, returns a dummy class with a placeholder method that does nothing,
+	allowing tests dependent on hypothesis to be safely bypassed. If True, the original
+	class is returned unchanged.
+	Args:
+		has_hypothesis (bool): Flag indicating whether the hypothesis library is available.
+	"""
```

Comment:
*NIT* - This can be improved further to align with project conventions and style;

Consider the following improvements:

```suggestion
	Conditionally enable a class based on the availability of the hypothesis library.

	If the provided flag is False, returns a dummy class with a placeholder method that does nothing,
	allowing tests dependent on hypothesis to be safely bypassed. If True, the original
	class is returned unchanged.

	Arguments:
		has_hypothesis (bool): Flag indicating whether the hypothesis library is available.

	Returns:
		callable: A decorator function that returns either the original class or a dummy class
		with a placeholder method, depending on the has_hypothesis flag.
	"""
```